src/index.tsx

```diff
@@ -65,6 +69,8 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({
 
   const setTheme = useCallback(
     theme => {
+      // When a theme is forced it should not be possible to override it.
+      if (forcedTheme) return
```

Comment:
Here's a tricky bug that I missed in 0.1.0 (and then fixed in 0.1.1):

1. Set theme to dark
2. Navigate to dark-forced page
3. In another tab, set theme to light (fires localStorage event)
4. The dark forced page receives the event (theme is now light)
4. Navigate back to non-forced page
5. Since we ignored the `setTheme` call, the theme is still considered forced/dark

Reply:
I think we'll need an extra piece of state (or maybe just a `ref`) to keep track of a "pending"/"upcoming" theme, so that when forcedTheme goes from `dark` → `undefined` we can do `setTheme(pendingTheme)` so everything is up-to-date.

Reply:
I added some more utilities to test storage-events. I'll implement the logic to handle this edge case and write the test later today.